_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1386,12 +1573,12 @@ Create all the corresponding input variables for `account-baseline-security` in
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "name_prefix" {
-  description = "The name used to prefix AWS Config and CloudTrail resources, including the S3 bucket names and SNS topics used for each."
+  description = "The name used to prefix AWS Config and Cloudtrail resources, including the S3 bucket names and SNS topics used for each."
```

Comment:
Why change "CloudTrail" to "Cloudtrail"? The former is correct.

Reply:
Copy/pasted from `module-security`. Fixed: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/344/commits/ee1c263846283740c72e99dd085b92198a2571be. Should probably fix the same in `module-security` itself.